eng/Publishing.props

```diff
@@ -1,5 +1,26 @@
 <Project>
-   <PropertyGroup>
-      <ProducesDotNetReleaseShippingAssets>true</ProducesDotNetReleaseShippingAssets>
-   </PropertyGroup>
-</Project>
\ No newline at end of file
+
+  <PropertyGroup>
+    <ProducesDotNetReleaseShippingAssets>true</ProducesDotNetReleaseShippingAssets>
+    <PublishDependsOnTargets Condition="'$(DotNetBuildRepo)' == 'true'">$(PublishDependsOnTargets);PublishWindowsDesktopInstallers</PublishDependsOnTargets>
+  </PropertyGroup>
+
+  <!-- Include installer archives and packages which aren't globbed by default. -->
+  <Target Name="PublishWindowsDesktopInstallers">
+    <!-- Retrieve windows desktop runtime pack product version. -->
+    <MSBuild Projects="$(RepoRoot)src/windowsdesktop/src/sfx/Microsoft.WindowsDesktop.App.Runtime.sfxproj"
```

Comment:
I think we need a different project which is non-shipping same as in aspnetcore: https://github.com/dotnet/aspnetcore/blob/6b7a6a94fb3ff2c004f354dbfe662644dac0ce26/eng/Publishing.props#L48-L49